src/v2/Apps/PriceDatabase/Components/ArtistAutosuggest.tsx

```diff
@@ -0,0 +1,165 @@
+import Autosuggest from "react-autosuggest"
+import React, { useEffect, useState } from "react"
+import {
+  Input,
+  MagnifyingGlassIcon,
+  usePosition,
+  Text,
+  Clickable,
+} from "@artsy/palette"
+import { graphql } from "lib/graphql"
+import { fetchQuery } from "relay-runtime"
+import { ArtistAutosuggest_SearchConnection_Query } from "v2/__generated__/ArtistAutosuggest_SearchConnection_Query.graphql"
+import { useSystemContext } from "v2/System"
+import { Container } from "v2/Components/Sticky"
+
+interface ArtistAutosuggestProps {
+  onChange?: (artustSlug: string) => void
+}
+
+export const ArtistAutosuggest: React.FC<ArtistAutosuggestProps> = ({
+  onChange,
+}) => {
+  const { relayEnvironment } = useSystemContext()
+
+  const [suggestions, setSuggestions] = useState<any>([])
+  const [searchQuery, setSearchQuery] = useState("")
+
+  const updateSuggestions = async () => {
+    const suggestions = await fetchSuggestions(searchQuery, relayEnvironment)
+
+    setSuggestions(suggestions)
+  }
+
+  useEffect(() => {
+    updateSuggestions()
+  }, [searchQuery])
+
+  const [width, setWidth] = useState(0)
+
+  useEffect(() => {
+    if (!anchorRef.current) return
+
+    const handleResize = () => {
+      setWidth(anchorRef.current.offsetWidth)
+    }
+
+    handleResize()
+
+    window.addEventListener("resize", handleResize)
+    return () => {
+      window.removeEventListener("resize", handleResize)
+    }
+  }, [])
+
+  const { anchorRef, tooltipRef } = usePosition({
+    position: "bottom",
+    offset: 10,
+  })
+
+  const selectSuggestion = async (selectedSuggestion: any) => {
+    onChange?.(selectedSuggestion?.node?.slug)
+  }
+
+  const AutosuggestInput: React.FC = props => {
+    return (
+      <>
+        <Input
+          width="100%"
+          spellCheck={false}
+          style={{ boxShadow: "0px 2px 10px rgba(0,0,0,0.1)" }}
+          {...props}
+          ref={anchorRef as any}
+        />
+        <MagnifyingGlassIcon position="absolute" right="2%" top={"16px"} />
+      </>
+    )
```

Comment:
We've got `LabelledInput` for these: https://palette-storybook.artsy.net/?path=/story/components-labeledinput--default